sql/core/src/test/resources/sql-tests/inputs/limit.sql

```diff
@@ -1,23 +1,27 @@
 
 -- limit on various data types
-select * from testdata limit 2;
-select * from arraydata limit 2;
-select * from mapdata limit 2;
+SELECT * FROM testdata LIMIT 2;
```

Comment:
Yes. This is preferred. All the SQL keywords are preferred to use the upper case.